proto/query.proto

```diff
@@ -986,8 +988,8 @@ message StreamHealthResponse {
 enum TransactionState {
   UNKNOWN = 0;
   PREPARE = 1;
-  COMMIT = 2;
-  ROLLBACK = 3;
+  ROLLBACK = 2;
+  COMMIT = 3;
```

Comment:
Why are we changing this? Is this required?

Reply:
This is to keep the order of state such that if we order on the transaction state, the commit state comes on top.